### ISSUE TITLE
warning message with EntityFilter on PartyRelationship

### ISSUE THREAD

**Issue by locnguyenx** [moqui/mantle-usl#43]
When call `service mantle.party.PartyServices.find#Party`, a warning message occurred
>Error adding authz entity filter condition: org.moqui.entity.EntityException: Tried to filter on field toPartyId which is not included in view-entity mantle.party.FindPartyView

In Mantle EntityFilter setup:
```
<filters entityFilterId="MANTLE_USER_ORG_8" entityName="mantle.party.PartyRelationship" filterMap="[_join:'or', _list:[[toPartyId:filterOrgIds, _comp:'in'], [relationshipTypeEnumId:(filterOrgIds ? 'PrtEmployee' : null), _comp:'!=']]]"/>
```

If I understood correctly, to inherit pre-defined filter setup on PartyRelationship, we should add alias `toPartyId` into view-entity definition FindPartyView and use exact name for relationshipTypeEnumId.

**Comment by jonesde** [comment]
I was unable to reproduce this issue. I added a 'Contact Owner' field to the party FindOptionsForm, now in PopCommerce on the FindCustomer and FindSupplier screens, and using it worked fine.

**Comment by locnguyenx** [comment]
I'm sorry about not describe in details about the situation.

I intend to use ArtifactAuthzFilter feature by doing following steps:

1. Assign filterSet into an artifactAuthzId by add following line in seed-data loading: 
```
<moqui.security.ArtifactAuthzFilter artifactAuthzId="POP_COMMERCE_ADM_ADM" entityFilterSetId="MANTLE_USER_ORG"/>
```
2.  Edit the root screen of the application, i.e in PopCommerceAdmin.xml and add this service call: 
```
<always-actions>
...
        <service-call name="mantle.party.PartyServices.setup#UserOrganizationInfo"/>
 </always-actions>
```
3. Open any screen that use find#Party, and I get that warning message in consol output

p/s: I've create new view and service based on FindPartyView and find#Party service, then add 2 more fields `toPartyId`, `relationshipTypeEnumId`. It works as expected.

**Comment by jonesde** [comment]
This is resolved in mantle-usl commit #af31a93. The real problem is the wrong field for the organization is used in the FindPartyView view-entity, the contact owner should be the toPartyId. I verified this works as the current authz entity filter code handles member-entity fields with a different alias name just fine.

Also see PopCommerce commit #2378339 where I made some changes for organization selection and filtering.
